src/helpers/tx.helper.js

```diff
@@ -69,7 +69,12 @@ async function getUtxosToFillTx(wallet, sumOutputs, options) {
 
   const { utxos } = utxosDetails;
   // Sort utxos with larger amounts first
-  utxos.sort((a, b) => b.amount - a.amount);
+  utxos.sort((a, b) => {
+    if (a.amount < b.amount) {
+      return 1;
+    }
+    return a.amount > b.amount ? -1 : 0;
+  });
```

Comment:
I had to change this because the output of `sort` expects a `Number`, so the previous code would fail. I also couldn't use nested ternary because the linter prevents it. But the behavior should be the same.

Reply:
Maybe we should add a test to this.
It would also increase the coverage of the PR

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-headless/pull/509/commits/9c1caefcf3b04b7b9ba371d35d77643a5d5a0338